content/vendor/guides/helm-chart.md

```diff
@@ -0,0 +1,119 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "Package a Helm Chart"
+weight: "1005"
+draft: true
+---
+
+In this guide, we'll explore how you can leverage software bundled using Helm to integrate an off-the-shelf application into your KOTS app bundle. We'll use the Consul helm chart as an example in this case.
+
+This guide will assume you've already completed one of the [Getting Started Guides](/vendor/guides/#getting-started) and set up a local git repo to manage your application. It is divided into four sections
```

Comment:
```suggestion
This guide will assume you've already completed one of the [Getting Started Guides](/vendor/guides/index#getting-started) and set up a local git repo to manage your application. It is divided into four sections
```

Reply:
I don't understand the need for this one -- I just went to https://kots.io/vendor/guides/#getting-started and it worked